build.gradle

```diff
@@ -10,15 +14,25 @@ repositories {
 }
 
 dependencies {
-    compile ('org.testng:testng:6.12')
+    compile('org.testng:testng:6.12',
+            'io.github.sskorol:test-data-supplier:1.3.0',
+            'org.projectlombok:lombok:1.16.18'
+    )
+    testCompile 'org.assertj:assertj-core:3.8.0'
 }
 
-task wrapper (type: Wrapper){
+task wrapper(type: Wrapper) {
     gradleVersion = '4.2.1'
 }
 
 test {
+//    testLogging {
```

Comment:
it's not a good practice to push commented code; would be better to remove, if it's not used